cmd/pluginctl/cmd/run.go

```diff
@@ -74,16 +79,16 @@ var cmdRun = &cobra.Command{
 				case watch.Deleted:
 					fallthrough
 				case watch.Modified:
-					job := event.Object.(*batchv1.Job)
-					if len(job.Status.Conditions) > 0 {
```

Comment:
Was this meant to check if the type assertion was successful? If so, I just used a type switch. If it was meant to check for additional conditions, I can add it back.

Reply:
I added the check back in, so please ignore this.